tasks.c

```diff
@@ -6992,27 +6996,29 @@ static void prvResetNextTaskUnblockTime( void )
 
     void vTaskEnterCritical( void )
     {
+        const BaseType_t xCoreID = ( BaseType_t ) portGET_CORE_ID();
```

Comment:
Similar reason above. Suggest to acquire xCoreID with interrupt disabled.

Reply:
Done!

Reply:
We suggest avoiding variable declarations before statements to comply with C89 standards.
I've also created a [pull request](https://github.com/felixvanoost/FreeRTOS-Kernel/pull/1/files) to your branch to address my suggestsions. Please help to take a look. Feel free to adapt it to your PR if you find it suitable.

Reply:
The changes seem reasonable to me, I've merged your PR into my branch.